XLForm/XL/Descriptors/XLFormRowDescriptor.m

```diff
@@ -136,6 +136,7 @@ -(XLFormBaseCell *)cellForFormController:(XLFormViewController * __unused)formCo
         NSBundle *bundle = [NSBundle mainBundle];
         NSString *cellClassString = cellClass;
         NSString *cellResource = nil;
+        NSBundle *bundleForCaller = [NSBundle bundleForClass:self.class];
```

Comment:
As far as I can tell, this is the XLForm bundle. Shouldn't this be `[NSBundle bundleForClass:cellClass]`? That would point to the caller's framework bundle.